src/azure-cli/azure/cli/command_modules/resource/_validators.py

```diff
@@ -31,12 +37,6 @@ def _validate_deployment_name(namespace):
             namespace.deployment_name = 'deployment1'
 
 
-def process_deployment_create_namespace(namespace):
```

Comment:
seems no change this file, can you revert this change?

Reply:
sure. updated.